lib/simulator-xcode-6.js

```diff
@@ -290,6 +291,10 @@ class SimulatorXcode6 extends EventEmitter {
     if (!this.connectHardwareKeyboard) {
       args.push('-ConnectHardwareKeyboard', '0');
     }
+
+    // Do the 'Touch ID Enroll' key bindings before the Simulator starts
+    await setTouchEnrollKey();
```

Comment:
Is it a desired capability or an endpoint or both? It seems like those two ought to be mutually exclusive (ie: either setting a desired capability `enrollTouchId=true` enables touch ID or calling the endpoint `/wd/hub/touch_id_enroll` toggles touchID enrollment).

If we use a desired capability, do we even need to be able to toggle touchID enrollment? Are there possible e2e test cases that would involve a device being enrolled in touchID and then unenrolled?